packages/g/glycin/package.yml

```diff
@@ -0,0 +1,30 @@
+name       : glycin
+version    : 0.1.2
+release    : 1
+source     :
+    - https://download.gnome.org/sources/glycin-loaders/0.1/glycin-loaders-0.1.2.tar.xz : c76c01925abd070174589ccb916a44a573ab65b1e9d493f154e42756431e6dd7
+homepage   : https://gitlab.gnome.org/sophie-h/glycin
+license    :
+    - LGPL-2.1-or-later
+    - MPL-2.0
+component  : desktop.gnome
+summary    : Sandboxed and extendable image decoding
+description: |
+    Glycin allows to decode images into gdk::Texture(s) and to extract image metadata. The decoding happens in sandboxed modular image loaders.
+builddeps  :
+    - pkgconfig(cairo)
+    - pkgconfig(gtk4)
+    - pkgconfig(libheif)
+    - pkgconfig(libxml-2.0)
+    - cargo
+    - git
+    - rust
```

Comment:
`rust` is pulled in by `cargo` and so you don't need to specify it manually

Reply:
will remove